newscoop/src/Newscoop/NewscoopBundle/Security/Http/Authentication/AuthenticationSuccessHandler.php

```diff
@@ -44,9 +46,10 @@ function onAuthenticationSuccess(Request $request, TokenInterface $token)
     {
         $user = $token->getUser();
 
+        // This should actually be handle by the AuthenticationFailedHandler
         if (!$user->isAdmin()) { // can't go into admin
-            $redirector = \Zend_Controller_Action_HelperBroker::getStaticHelper('redirector');
-            $redirector->direct('index', 'index', 'default');
+            $request->getSession()->set(SecurityContextInterface::AUTHENTICATION_ERROR, new AuthenticationException('User is not an admin.'));
+            return $this->httpUtils->createRedirectResponse($request, 'admin_login');
```

Comment:
well, it suppose to redirect user to login page when user is not admin (dont have access to backend), @m038 am I right ?